Sources/Vortex/Views/VortexView.swift

```diff
@@ -55,6 +56,32 @@ public struct VortexView<Symbols>: View where Symbols: View {
         self.symbols = symbols()
     }
 
+    /// Creates a new VortexView from a pre-configured particle system, along with any required  SwiftUI
+    /// views needed to render particles. Sensible defaults will be used if no parameters are passed.
+    /// - Parameters:
+    ///   - settings: A vortexSettings struct that should be used to generate a particle system. 
+    ///               Typically this will be set using a preset static struct, e.g. `.fire`. Defaults to a simple system.
+    ///   - targetFrameRate: The ideal frame rate for updating particles. Defaults to 60 if not specified. (use 120 on Pro model iPhones/iPads )
+    ///   - symbols:  A closure that should return a tagged group of SwiftUI views to use as particles. 
+    ///               If not specified, a default group of three views tagged with `.circle`,`.confetti` and `.sparkle` will be used.
+    public init(
+        settings: VortexSystem.Settings = .init(),
+        targetFrameRate: Int = 60, 
+        @ViewBuilder symbols: () -> Symbols = {
+            Group {
+        Image.circle
```

Comment:
I appreciate the problem this is trying to solve, but I'm not sure it's really that practical – using images rather than shapes has a significant performance impact, and each system uses different variations. For example, `.confetti` adds a square shape and doesn't use plus lighter, `.rain` uses 32-point circles without plus lighter, fire adds a blur over 32-point circle, etc.